### PR TITLE
more typo fixes

### LINKED ISSUE #62667: Minor typo - Magiclysm
### Describe the bug

The description of the "Freeze burn" effect type in Magiclysm has a minor typo, said description for context:
- You skin is frozen.
This should be:
- Your skin is frozen.

Pretty minor typo, but it bugged me out while translating the string

### Steps to reproduce

1. Launch the game
2. Apply the effect to yourself/some creature
3. Check its description

### Expected behavior

Not having the typo.

### Screenshots

_No response_

### Versions and configuration

Not actually checked in game, but as Transifex pulls strings from the latest build of experimental, it is to be assumed it will be a thing in said latest build of experimental.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by casswedson**
#### Summary
None

#### Purpose of change
fix #62667

it all started with just a little typo, 3 hours later I reviewed and passed all magiclysm's descriptions through [LanguageTool](https://languagetool.org/)

#### Testing
string editing

### CODE REVIEW

File: data/mods/Magiclysm/Spells/attunements/Stormcaller.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "id": "electric_turret",
     "type": "SPELL",
     "name": { "str": "Thunder Pillar" },
-    "description": "Summon an immovable pillars, that will attack your enemies with devastating lightnings.",
+    "description": "Summon immovable pillars, that will attack your enemies with devastating lightnings.",
```
Comment (NetSysFire) [inline data/mods/Magiclysm/Spells/attunements/Stormcaller.json:37]:
```suggestion
    "description": "Summon immovable pillars, that will attack your enemies with devastating lightning bolts.",
```

File: data/mods/Magiclysm/Spells/black_dragon.json [hunk 2]
```diff
@@ -196,7 +196,7 @@
     "id": "dragon_spit_acid",
     "type": "SPELL",
     "name": "Dragon Acid Spit",
-    "description": "You conjur up acidic spittle that you can shoot long distances as you close on your terrified prey.",
+    "description": "You conjure up acidic spittle that you can shoot long distances as you close on your terrified prey.",
```
Comment (NetSysFire) [inline data/mods/Magiclysm/Spells/black_dragon.json:199]:
```suggestion
    "description": "You conjure up acidic spittle that you can shoot long distances as you close in your terrified prey.",
```

File: data/mods/Magiclysm/Spells/item_only.json [hunk 3]
```diff
@@ -409,7 +409,7 @@
     "id": "flask_teleport",
     "type": "SPELL",
     "name": "Ecfugium",
-    "description": "Teleport the caster to picked point nearby.",
+    "description": "Teleport the caster to a picked point nearby.",
```
Comment (NetSysFire) [inline data/mods/Magiclysm/Spells/item_only.json:412]:
```suggestion
    "description": "Teleport the caster to a selected point nearby.",
```

File: data/mods/Magiclysm/Spells/item_only.json [hunk 4]
```diff
@@ -541,7 +541,7 @@
     "id": "flask_speed",
     "type": "SPELL",
     "name": "Velocitas",
-    "description": "Make you faster, and allow to hold this state for a while.",
+    "description": "Makes you faster, and allow to hold this state for a while.",
```
Comment (NetSysFire) [inline data/mods/Magiclysm/Spells/item_only.json:544]:
```suggestion
    "description": "Makes you faster and allows you to be in this state for a while.",
```

**Review by NetSysFire — COMMENTED** [review]
Not even done yet, some of these descriptions are super flaky. Thanks to your contributions they are a bit better now but I feel like they will need some rewording in the future.

**Comment by casswedson** [comment]
I would say I have a hard time with words english in general, more in topic with this game descriptions, I understand how hard it is to get them right

**Comment by NetSysFire** [comment]
To be clear: My comment is about the overall state of the descriptions in the files you modified, not your typo fixes. Thanks for taking the time to make them better by fixing the typos.

File: data/mods/Magiclysm/Spells/black_dragon.json [hunk 5]
```diff
@@ -196,7 +196,7 @@
     "id": "dragon_spit_acid",
     "type": "SPELL",
     "name": "Dragon Acid Spit",
-    "description": "You conjur up acidic spittle that you can shoot long distances as you close on your terrified prey.",
+    "description": "You conjure up acidic spittle that you can shoot long distances as you close in your terrified prey.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/Spells/black_dragon.json:199]:
```suggestion
    "description": "You conjure up acidic spittle that you can shoot long distances as you close in on your terrified prey.",
```

File: data/mods/Magiclysm/items/cast_spell_items.json [hunk 6]
```diff
@@ -103,7 +103,7 @@
     "id": "flask_confusion",
     "type": "COMESTIBLE",
     "name": { "str": "potion of instability", "str_pl": "potions of instability" },
-    "description": "That thing temporary freeze the high brain activity for a while.  It was banned by the government due the popularity in crime sphere, but some people still know the recipe.",
+    "description": "That thing temporary freeze the high brain activity for a while.  It was banned by the government due to the popularity in crime sphere, but some people still know the recipe.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/items/cast_spell_items.json:106]:
```suggestion
    "description": "A potion that temporarily freezes the higher brain activity for a while.  It was banned by the government due to popularity in the criminal sphere, but some people still know the recipe.",
```

File: data/mods/Magiclysm/items/cast_spell_items.json [hunk 7]
```diff
@@ -123,7 +123,7 @@
     "id": "flask_teleport",
     "type": "COMESTIBLE",
     "name": { "str": "potion of teleportation", "str_pl": "potions of teleportation" },
-    "description": "Important part of every professional robber.  It was banned by the government due the popularity in crime sphere, but some people still know the recipe.",
+    "description": "Important part of every professional robber.  It was banned by the government due to the popularity in crime sphere, but some people still know the recipe.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/items/cast_spell_items.json:126]:
```suggestion
    "description": "Important part of every professional robber's kit.  It was banned by the government due to the popularity in criminal sphere, but some people still know the recipe.",
```

**Comment by Maleclypse** [comment]
Sorry I kept finding sentences that I was like "I can improve this quickly".

**Comment by casswedson** [comment]
no problem I do that all the time

**Pull request merged**